lib/engine/game/g_1880/game.rb

```diff
@@ -650,6 +664,29 @@ def check_for_foreign_investor_connection(fi)
           graph = Graph.new(self, no_blocking: true)
           graph.reachable_hexes(corporation).include?(fi_home_token)
         end
+
+        def after_par(corporation)
+          super
+          @par_order[corporation.par_price.price] << corporation
+        end
```

Comment:
parred companies are auto allocated to the top available position in each par price, this will change, We're in the middle of implementing a UI for picking exact par order spot